internal/tableprinter/render_json.go

```diff
@@ -0,0 +1,39 @@
+package tableprinter
```

Comment:
Can you please put the license header. This will be the last requested change :)

```diff
+ // SPDX-License-Identifier: BSD-3-Clause
+ // Copyright (c) 2022, Unikraft GmbH and The KraftKit Authors.
+ // Licensed under the BSD-3-Clause License (the "License").
+ // You may not use this file expect in compliance with the License.
```

Reply:
done 👍🏽